src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -88,6 +88,14 @@ module.exports = React.createClass({
         if (this.props.onFinished) this.props.onFinished();
     },
 
+    onForwardClick: function() {
+        dis.dispatch({
+            action: 'forward_message',
+            content: this.props.mxEvent.getContent(),
```

Comment:
its currently forward_message, will be extended to forward_event, not for state events and maybe some others blacklisted and whatnot if the need is there